README.md

```diff
@@ -1 +1,139 @@
-# BooksStore
\ No newline at end of file
+# 📗 Table of Contents
+
+<h3><b>Calculator README Template</b></h3>
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Calculator\] ](#-calculator-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Run test](#run-test)
+    - [Run App](#run-app)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 [Calculator] <a name="about-project"></a>
```

Comment:
Great job @hasnatali1947 🤝

- [ ] Kindly ensure your readme file references the actual project you are working on. Make sure you are referencing BooksStore instead of Calculator.